### DISCUSSION TITLE
Is there a corresponding specification for the installation height of UWB ANCHEOR?

### DISCUSSION THREAD

**Discussion by lalalalang999** [bitcraze/discussions#676] [Q&A]
In the UWB construction document, I saw that the UWB ANCHOR recommended method is an 8-base station system. Install 4 anchors above the flying space and 4 anchors on the ground. In the process of my actual use, since I don't need the data of the Z axis, I installed 8 anchors above the flight space, and no anchors were installed on the ground. For the installation of 8 anchors directly above, there must be differences in the heights between the anchors, or must the heights of the 8 anchors be the same.

**Comment by krichardsson** [comment]
For 2D positioning, the optimal height of the anchors is the same height as the Crazyflie. It should not be a problem if you need to put them a bit higher, but too high is less good. There is no need to put the anchors on different heights in 2D positioning. 

For 2D positioning, you should compile your own firmware for 2D positioning. Configure by running `make menuconfig`
1. Go to `Expansion deck configuration`
2. Go to `Support the Loco positioning deck`
3. Set `Algorithm to use` to `Use the Time Difference of Arrival (3) algorithm`
4. Select `If set we assume we are doing 2D positioning`
5. Set `The height (Z) that the tag will be located at` to the height you will use the Crazyflie on
